policyengine_us/parameters/gov/states/ia/tax/income/deductions/itemized/applies_federal.yaml

```diff
@@ -0,0 +1,14 @@
+description: Iowa applies the federal itemized deduction if this is true.
+values:
+  0000-01-01: false
+  2023-01-01: true
+
+metadata:
+  label: Iowa federal itemized deduction applies
+  period: year
+  unit: bool
+  reference: 
+  - title: Iowa Department of Revenue - 2023 IA1040 Income Tax Return instructions - Iowa Taxable Income
+    href: https://revenue.iowa.gov/taxes/tax-guidance/individual-income-tax/1040-expanded-instructions/iowa-taxable-income
+  - title: Iowa Individual Income, Corporate, and Franchise Taxes, §422.5 3(a)
```

Comment:
I don't see this here. It talks about adding federal deductions back for the purposes of, I think, some state tax deduction? Can you explain in a comment if it's here?

Reply:
You are right, 

I found the legal code but it does not seem to [represent the changes](https://casetext.com/regulation/iowa-administrative-code/agency-701-revenue-department/title-iv-individual-income-tax/chapter-303-determination-of-taxable-income/rule-701-3034-optional-standard-deduction) or at least explicitly state the change in 2023 - have added a comment - the instructions are clear that the federal ded should be used

Reply:
where does the law define the standard deduction values pre-2023?